$OpenBSD: patch-engine_SCons_Tool_m4_py,v 1.1 2011/12/21 17:41:52 jasper Exp $

XXX: OpenBSD's m4(1) lacks the -E option (needs to be implemented though).

--- engine/SCons/Tool/m4.py.orig	Wed Dec 21 18:39:51 2011
+++ engine/SCons/Tool/m4.py	Wed Dec 21 18:40:07 2011
@@ -50,7 +50,6 @@ def generate(env):
     # The src_suffix setup is like so: file.c.m4 -> file.c,
     # file.cpp.m4 -> file.cpp etc.
     env['M4']      = 'm4'
-    env['M4FLAGS'] = SCons.Util.CLVar('-E')
     env['M4COM']   = 'cd ${SOURCE.rsrcdir} && $M4 $M4FLAGS < ${SOURCE.file} > ${TARGET.abspath}'
 
 def exists(env):
